### PR TITLE
docs: remove SCSS from the landing page, update library name

### PR DESCRIPTION
**Pull request by droshev**
fixes: #427

### CODE REVIEW

**Comment by bcullman** [comment]
date picker & multiimput are 404/ navigating to wrong locations in patterns section:

https://deploy-preview-429--fundamental-styles.netlify.com/patterns/multi-input.html

File: docs/_data/sidebars/left-navigation-sidebar.yml [hunk 1]
```diff
@@ -61,17 +61,21 @@ entries:
...
+      output: web
   
     - title: Contextual Menu
       url: /patterns/contextual-menu.html
       output: web
 
-    - title: Combobox Input
-      url: /patterns/combobox-input.html
+    - title: Date Picker
+      url: /patterns/multi-input.html
```
Comment (jbadan) [inline docs/_data/sidebars/left-navigation-sidebar.yml]: I think this is the cause of the 404

File: docs/_includes/display-component.html [hunk 2]
```diff
@@ -1,14 +1,4 @@
-{% assign min = 50 %}
-{% assign max = 100000 %}
-{% assign diff = max | minus: min %}
-{% assign randomNumber = "now" | date: "%N" | modulo: diff | plus: min %}
 <div class="fd-tile docs-component docs-component__{{include.class}}">
-    <label class="fd-form-label fd-form-label--toggle docs-component__bg-toggle" for="{{ randomNumber }}" title="Change Background">
-        <span class="fd-toggle fd-toggle--xs">
-            <input class="toggle-bg fd-toggle__input" type="checkbox" name="" value="" id="{{ randomNumber }}">
-            <span class="fd-toggle__switch" role="presentation"></span>
-        </span>
```
Comment (jbadan) [inline docs/_includes/display-component.html]: Why are we getting rid of the background toggle?

Comment (droshev) [reply docs/_includes/display-component.html]: What is the actual value of this?

File: docs/_data/sidebars/left-navigation-sidebar.yml (hunk 1 above)
Comment (droshev) [reply docs/_data/sidebars/left-navigation-sidebar.yml]: updated

File: config/postcss.config.js [hunk 3]
```diff
@@ -24,8 +24,8 @@ module.exports = {
     sourcesContent: true,
     plugins: [
         postcssImport(),
-        postcssAddFallback({importFrom: 'dist/root.css'}),
-        postcssAddFallback({importFrom: 'node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/css_variables.css'}),
+        postcssAddFallback({ importFrom: 'dist/root.css' }),
+        postcssAddFallback({ importFrom: 'node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/css_variables.css' }),
```
Comment (InnaAtanasova) [inline config/postcss.config.js]: Could you revert the re-formatting?

Comment (droshev) [reply config/postcss.config.js]: reverted

**Pull request merged**